lib/ai/prompt.go

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2023 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai
+
+import "fmt"
+
+const promptSummarizeTitle = `You will be given a message. Create a short summary of that message.
+Respond only with summary, nothing else.`
+
+const initialAIResponse = `Hey, I'm Teleport - a powerful tool that can assist you in managing your Teleport cluster via ChatGPT.`
+
+const promptExtractInstruction = `If the input is a request to complete a task on a server, try to extract the following information:
```

Comment:
I also vote for "extract", but @xacrimon is the right person to ask.

Reply:
I recall changing it to "try" instead of a plain "extract" because sometimes it would complain audibly that it couldn't do the thing and it totally broke the immersionl.